tests/tests/regional_spp_sppt_shum_skeb

```diff
@@ -0,0 +1,60 @@
+####################################################################
+#
+#  Regional test with stochastic physics (SPP, SPPT, SHUM, and SKEB)
+#  
+####################################################################
+
+export TEST_DESCR="Compare regional stochastic physics results with previous trunk version"
+
+export CNTL_DIR=regional_spp_sppt_shum_skeb
+
+export LIST_FILES="dynf000.nc \
+                   dynf001.nc \
+                   phyf000.nc \
+                   phyf001.nc \
+                   PRSLEV.GrbF00 \
+                   PRSLEV.GrbF01 \
+                   NATLEV.GrbF00 \
+                   NATLEV.GrbF01"
+
+export_fv3
+
+export FV3_RUN=regional_run_stoch.IN
```

Comment:
I do not see this file added to this branch.

Reply:
Added in latest commit.

Reply:
Added this file in the latest commit.